### PR TITLE
Added implementation for Docker in Hyper containers

### PR DESCRIPTION
**Pull request by LFlare**
This will allow for building docker images on hyper containers. There has to be a better way of implementing this with `plugins/docker` but I have no idea what. Would appreciate feedback from you @bradrydzewski.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -17,6 +17,16 @@ pipeline:
       branch: master
       event: push
 
+  publish:
```
Comment (tboerger) [inline .drone.yml]: Wrong step name

File: .drone.yml [hunk 2]
```diff
@@ -17,6 +17,16 @@ pipeline:
...
+    image: plugins/docker
+    repo: plugins/docker
```
Comment (tboerger) [inline .drone.yml]: I don't think that brad want's to add this to the same image as this is generally feature-frozen.

**Comment by LFlare** [comment]
@tboerger, thanks for the feedback, I made some typos and after some considerations, I think `plugins/hyper` is a better choice, what do you think?

**Comment by bradrydzewski** [comment]
I recommend starting by creating your own `LFlare/hyper` image. The process we generally following is that all plugins __must__ incubate as third party plugins before we would consider adding the the official plugins organization.

**Comment by LFlare** [comment]
Alright, I've done it through `LFlare/drone-hyper` for now.

**Pull request closed without merging**